core/shared/src/main/scala/zio/ZLayer.scala

```diff
@@ -2034,6 +2040,14 @@ object ZLayer {
     ): ZLayer[RIn with RIn2, E1, ROut with ROut2] =
       self.zipWithPar(that)(_.unionAll[ROut2](_))
 
+    /**
+      * A named alias for `++`.
+      */
+    def and[E1 >: E, RIn2, ROut1 >: ROut, ROut2 <: Has[_]](
```

Comment:
Not to delay further but I'm not sure they are alphabetized. I'll approve though!

Reply:
I think they are but `and` just appears far down in the file because it is in the syntax class that provides evidence that the output type is a subtype of `Has[_]`.